client/src/components/QandA/AddAnswer.jsx

```diff
@@ -35,34 +69,43 @@ function AddAnswer({
     return null;
   }
   return (
+    // eslint-disable-next-line
     <div className="modal" onClick={() => setShow(!show)}>
+      {/* eslint-disable-next-line */}
       <div className="modal-content" onClick={(e) => e.stopPropagation()}>
         <div className="modal-header">
           <h4 className="modal-title">Submit your Answer</h4>
-          <h5 className="modal-subtitle">
+          <div className="modal-subtitle">
             {product.name}
             :
             {' '}
             {question.question_body}
             {' '}
-          </h5>
+          </div>
         </div>
         <form onSubmit={handleSubmit} className="modal-body">
-          <div className="modalBodyTitle">Your answer: </div>
```

Comment:
I recently just learned about this but html classname naming conventions should be all lowercase or separated by hyphens (-). 
Eg. 'modalbodytype' or 'modal-body-type'